src/content/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/rest-api-calls-alerts.mdx

```diff
@@ -783,7 +783,7 @@ These API functions include links to the API Explorer, where you can create, upd
         },
         "signal": {
     	  "<a href="/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/alerts-conditions-api-field-names#signal_aggregation_window">aggregation_window</a>": "string",
-    	  "<a href="/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/alerts-conditions-api-field-names#signal_slideby">slideby</a>": "integer",
+    	  "<a href="/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/alerts-conditions-api-field-names#signal_slideby">slide_by</a>": "integer",
```

Comment:
I don't think this link is working correctly. When I mouse over the anchor in the field names doc it says "/docs/alerts-applied-intelligence/new-relic-alerts/advanced-alerts/rest-api-alerts/alerts-conditions-api-field-names#signal_slide_by" not "signal_slideby"

Reply:
The link works, I just changed the link text. I'll update the URL too.

Reply:
It looked like the URL changed too, not just the link text

Reply:
Nevermind, I see you are on the case!